src/status_im/config.cljs

```diff
@@ -63,7 +64,11 @@
 (def show-not-implemented-features? (enabled? (get-config :SHOW_NOT_IMPLEMENTED_FEATURES "0")))
 
 ;; CONFIG VALUES
-(def log-level (string/upper-case (get-config :LOG_LEVEL "")))
+(defn log-level
+  []
+  (if (mmkv/contains-key? constants/pre-login-log-level-key)
+    (mmkv/get-string constants/pre-login-log-level-key)
+    (string/upper-case (get-config :LOG_LEVEL ""))))
```

Comment:
Can we use (mmkv/getstring key fallback) instead?

Reply:
Thank you, it looks better. It's still strange that we're not using its default value feature, especially since the function supports it.